modules/ximgproc/samples/find_ellipses.cpp

```diff
@@ -0,0 +1,49 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+#include <iostream>
+#include <opencv2/imgproc.hpp>
+#include <opencv2/imgcodecs.hpp>
```

Comment:
#include <opencv2/highgui.hpp> needed

Reply:
Highgui dependency for tests should be optional. It's not available on many platforms.

Reply:
This is sample, not a test.